docroot/.htaccess

```diff
@@ -1,182 +0,0 @@
-#
```

Comment:
Intentionally removed this file?

Reply:
Yes- removing due to it being controlled by .htaccess and to avoid collisions.